<!-- Launch in the camera namespace containing "image_raw" and "camera_info" -->
<launch>

  <arg name="manager" /> <!-- Must be globally qualified -->
  <arg name="respawn" default="false" />
  <!-- TODO Arguments for debayer, interpolation methods? -->

  <arg     if="$(arg respawn)" name="bond" value="" />
  <arg unless="$(arg respawn)" name="bond" value="--no-bond" />

<!--  <arg name="background_image_path" default="$(find mmf_writer)/background.png" /> -->

  <node pkg="nodelet" type="nodelet" name="mmf_writer"
        args="load mmf_writer/mmf_writer_nodelet $(arg manager) $(arg bond)"
        respawn="$(arg respawn)">

<!--    <param name="background_image_path" value="$(arg background_image_path)"/> -->

    <!-- remap input topics -->
    <remap from="camera/image_raw" to="image_raw"/>
    <remap from="camera/image" to="image"/>
    <remap from="camera/camera_info" to="camera_info"/>

   </node>

</launch>
